scripts/run_script_2_clean_data.R

```diff
@@ -7,8 +7,8 @@ ERROR_VAL_DATE <<- "9999-09-09"
 main <- function() {
     paths <- init_paths(c("patient_data_cleaned", "product_data_cleaned"), delete = TRUE)
     setup_logger(paths$output_root, "script2")
-    patient_data_files <- get_files(paths$tracker_root, pattern = "patient_raw.csv$")
-    product_data_files <- get_files(paths$tracker_root, pattern = "product_raw.csv$")
+    patient_data_files <- get_files(paths$tracker_root, pattern = "patient_raw.parquet$")
```

Comment:
I guess I made an error here as pattern is a regex and "." should be "\\." as it has to be escaped like I did in the other file below, but yeah...works in this case anyway